assets/js/base/components/chip/chip.tsx

```diff
@@ -1,32 +1,30 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-/** @typedef {import('react')} React */
+export interface ChipProps {
```

Comment:
Here again perhaps it's worth it to preserve the docs? Especially for things that are not super clear immediately like `element`.